website/frontend/static/less/site-wide.less

```diff
@@ -21,24 +22,22 @@ a
 .navbar
 {
    margin: 0px;
-   border-width: 0 0 5px;
+   border-color: #fff;
    .navbar-brand
    {
       font-size: 20px;
    }
+   .nav{
+      border-bottom: 2px solid @mb-purple;
+   }
 }
 
+
 .jumbotron {
    padding: 15px;
-   color: #FFF;
-   text-shadow: 0px 1px 3px rgba(0, 0, 0, 0.5), 0px 0px 30px rgba(0, 0, 0, 0.075);
-   box-shadow: 0px 3px 7px rgba(0, 0, 0, 0.25) inset, 0px -3px 7px rgba(0, 0, 0, 0.25) inset;
-   background: linear-gradient(0deg, #3A3756 0%, #736DAB 100%) repeat scroll 0% 0% transparent;
```

Comment:
I wonder if it would be better to just have horizontal line at the bottom instead of grey background.